langchain4j-astradb/src/main/java/dev/langchain4j/store/embedding/astradb/AstraDBEmbeddingStore.java

```diff
@@ -0,0 +1,324 @@
+package dev.langchain4j.store.embedding.astradb;
+
+import com.datastax.astra.client.Collection;
+import com.datastax.astra.client.model.DataAPIKeywords;
+import com.datastax.astra.client.model.Document;
+import com.datastax.astra.client.model.Filter;
+import com.datastax.astra.client.model.FindOneAndReplaceOptions;
+import com.datastax.astra.client.model.FindOptions;
+import com.datastax.astra.client.model.InsertManyOptions;
+import com.datastax.astra.client.model.Projections;
+import dev.langchain4j.data.document.Metadata;
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.store.embedding.EmbeddingMatch;
+import dev.langchain4j.store.embedding.EmbeddingSearchRequest;
+import dev.langchain4j.store.embedding.EmbeddingSearchResult;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static com.datastax.astra.client.model.Filters.eq;
+
+/**
+ * Implementation of {@link EmbeddingStore} using AstraDB.
+ *
+ * @see EmbeddingStore
+ */
+@Slf4j
+@Getter @Setter
+@Accessors(fluent = true)
+public class AstraDBEmbeddingStore implements EmbeddingStore<TextSegment> {
```

Comment:
```suggestion
public class AstraDbEmbeddingStore implements EmbeddingStore<TextSegment> {
```
Please use AstraDb naming everywhere, to keep the same style as everywhere else in lc4j

Reply:
ok the rational was the product name itself is `AstraDB`, changing to be nature